components/02-elements/price/price.config.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  name: 'price',
+  context: {
+    regularPrice: '1 400',
+    specialPrice: '',
+    prefix: {
+      tag: '',
+      closingTag: '',
```

Comment:
It's better to add one and write `<>` and `</>` in template.

Reply:
sure. It's just I didn't want to use if tag as it could be optional to insert or not to insert tag ;) changed